django-stubs/contrib/admin/sites.pyi

```diff
@@ -61,11 +61,11 @@ class AdminSite:
     def is_registered(self, model: type[Model]) -> bool: ...
     def add_action(self, action: _ActionCallback, name: str | None = ...) -> None: ...
     def disable_action(self, name: str) -> None: ...
-    def get_action(self, name: str) -> Callable: ...
+    def get_action(self, name: str) -> _ActionCallback: ...
     @property
     def actions(self) -> Iterable[tuple[str, _ActionCallback]]: ...
     def has_permission(self, request: HttpRequest) -> bool: ...
-    def admin_view(self, view: Callable, cacheable: bool = ...) -> Callable: ...
+    def admin_view(self, view: Callable[..., HttpResponse], cacheable: bool = ...) -> Callable[..., HttpResponse]: ...
```

Comment:
Hm, `admin_view` looks like a decorator. I think we want to use a `TypeVar` to not ruin any function signature using it. e.g.

```suggestion
    def admin_view(self, view: F, cacheable: bool = ...) -> F: ...
```

Where `F` could be a variable bound to a callable returning a `HttpResponse`

```python
F = TypeVar("F", bound=Callable[..., HttpResponse])
```